src/pages/link-app/index.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  // AspectRatio,
+  Button,
+  Container,
+  createStyles,
+  Flex,
+  Grid,
+  Stack,
+  Text,
+  Title,
+  Image,
+} from '@mantine/core';
+// import { YoutubeEmbed } from '~/components/YoutubeEmbed/YoutubeEmbed';
+import { containerQuery } from '~/utils/mantine-css-helpers';
+import { useCurrentUser } from '~/hooks/useCurrentUser';
+import { NextLink } from '@mantine/next';
+import Lottie from 'react-lottie';
+import * as linkAnimation from '~/utils/lotties/link-animation.json';
+
+export default function LinkApp() {
+  const { classes } = useStyles();
+  const currentUser = useCurrentUser();
+  const isMember = currentUser?.isMember;
+  const buttonData = {
+    text: isMember ? 'Download the Link App' : 'Become a member',
+    href: isMember ? 'https://github.com/civitai/civitai-link-desktop/releases/latest' : '/pricing',
+  };
+
+  return (
+    <Container>
+      <Flex
+        direction={{ base: 'column', md: 'row' }}
+        justify="space-between"
+        className={classes.heroContainer}
+      >
+        <Stack spacing={12} mb={{ base: 24, md: 0 }}>
+          <Title className={classes.heroTitle} order={1}>
+            Civitai Link App
+          </Title>
+          {!isMember ? (
+            <Text className={classes.heroText}>❤️ Civitia Link is only available to members</Text>
+          ) : null}
+        </Stack>
+        <Flex align="center">
+          <Button
+            variant="filled"
+            color="blue"
+            size="lg"
+            radius="xl"
+            fullWidth
+            component={NextLink}
+            href={buttonData.href}
```

Comment:
Might be a good idea to add `rel` prop
```suggestion
            href={buttonData.href}
            rel="nofollow noreferrer"
```